### PR TITLE
Erase the Support Message for EDP

### PR DESCRIPTION
**Pull request by vverbani**
### **User description**
The previous message about contact support to get access is false. We don't give anyone access - especially when it comes  to the EDP. So getting rid of this message as it's false advertising. Please approve.

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

___

### **PR Type**
documentation

___

### **Description**
- Removed a misleading support contact message from the installation guide of the Tyk Enterprise Developer Portal.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>install-tyk-enterprise-portal.md</strong><dd><code>Remove Misleading Support Contact Message</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/install-tyk-enterprise-portal.md
<li>Removed a note block that falsely advertised contacting support for <br>access to Tyk Enterprise Developer Portal.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4628/files#diff-c8a340a0905207585ee5e2c0f912dd3bbd8f7a1510b9d9c8266cbace569c7625">+0/-8</a>&nbsp; &nbsp; &nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/install-tyk-enterprise-portal.md [hunk 1]
```diff
@@ -12,14 +12,6 @@ aliases:
...
 ---
 
-{{< note success >}}
-**Tyk Enterprise Developer Portal**
-
-If you are interested in getting access contact us at [support@tyk.io](<mailto:support@tyk.io?subject=Tyk Enterprise Developer Portal Beta>)
-
-{{< /note >}}
-
-## Installing Tyk Enterprise Developer Portal
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/install-tyk-enterprise-portal.md:22]: If we remove this heading then bump the style for the sub heading _Installation steps_ up a level, e.g. _###  Installation steps_ becomes _## Installation steps_

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks @vverbani PR LGTM. I have made minor suggestion relating to removal of Installing Tyk Enterprise Developer Portal:

If we remove this heading then bump the style for the sub heading Installation steps up a level, e.g. ### Installation steps becomes ## Installation steps

**Review by slega — APPROVED** [review]
LGTM

**Review by dcs3spp — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]
PR LGTM, I have updated heading style for Installation steps to be H2. Thanks @vverbani

**Pull request merged**